### DISCUSSION TITLE
What specific version of Proton 7.0 is Heroic Launcher using?

### DISCUSSION THREAD

**Discussion by msmirle** [Heroic-Games-Launcher/HeroicGamesLauncher#1731] [General]
On my Steam Deck, out of the box I see in the Heroic Launcher that there are two options for Proton.....Proton-Experimental and Proton 7.0.

If I select Proton 7.0, what specific version of 7.0 is being run? Is it the latest 7.0-3? Or something earlier? Thanks in advance!

**Comment by flavioislima** [comment]
It is the one installed on Steam. Heroic just reads the Steam directory and uses the ones there.

**Reply by msmirle** [reply]
Ah thanks very much! That makes a lot of sense and also explains why Heroic Launcher only offers options to download versions of WINE-GE, WINE-LUTRIS, and PROTON-GE as opposed to the vanilla versions of Proton that appear in Steam.
